aws/keyspaces/keyspaces.go

```diff
@@ -42,39 +41,36 @@ func SetLogger(l Logger) {
 	ksLogger = l
 }
 
-// New returns a Keyspaces struct which wraps a Keyspaces session. certPath
-// is the path of the required Starfield digital certificate to connect
+// New returns a Keyspaces struct which wraps a Keyspaces session.
+// host to connect to eg: cassandra.ap-southeast-2.amazonaws.com:9142
+// certPath is the path of the required Starfield digital certificate to connect
 // to Keyspaces using SSL/TLS
 // (found here: https://certs.secureserver.net/repository/sf-class2-root.crt)
-func New(certPath string) (Keyspaces, error) {
+func New(host, certPath string) (Keyspaces, error) {
```

Comment:
question: what should `certPath` be if I'm calling `New` with `"localhost"`?

Reply:
It's not used, so can be `""`.

Reply:
I thought as much. Lack of function overloading and default values is annoying here.